python/ray/tests/test_metrics.py

```diff
@@ -105,6 +108,7 @@ def f(self):
         if len(reply.core_workers_stats) < num_cpus + 2:
             time.sleep(1)
             reply = try_get_node_stats()
+            print(reply)
```

Comment:
hmm do we need this line?

Reply:
ah this is for debugging. I think it is okay to keep it

Reply:
successful tests' logs are not printed anyway